Code/Framework/AzCore/Tests/Serialization/Json/JsonSerializerConformityTests.h

```diff
@@ -17,6 +17,8 @@
 #include <AzCore/std/typetraits/is_base_of.h>
 #include <Tests/Serialization/Json/BaseJsonSerializerFixture.h>
 
+#define RESTRICT_JSON_CONFORMITY_TESTS(X) 
+
```

Comment:
This is disabling all of the JSON Serialization Conformity Test which should be done.

Reply:
Checked in in the wrong testing state.

Reply:
Ok, that clarifies it